contrib_bots/lib/hangman.py

```diff
@@ -0,0 +1,142 @@
+        self.output = "You are supposed to guess a letter, not a word! To guess again, type @hangman guess <letter>."
+    elif split[2] in self.guessed:
+        self.output = "You have already guessed the letter " + split[2] + "! To guess again, type @hangman guess <letter>."
+    else:
+        self.guessed.append(split[2])
+        if split[2] not in self.currentword:
+            self.stage += 1
+            if self.stage == 10:
+                end = "\n·\n:crying_cat_face: Oh no, you lost! The correct word was " + str(self.currentword) + "." \
+                      " To play again, type @hangman new."
+                self.ended = True
+        stages = ["", "·\n·\n·\n·\n·\n·\n=========",
+                  ("·\n···············|\n···············|\n···············|" +
+                   "\n···············|\n···············|\n========="),
+                  ("······+-----+\n···············|\n···············|\n···············|" +
+                   "\n···············|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n···············|\n···············|" +
+                   "\n···············|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n······0·······|\n···············|" +
+                   "\n···············|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n······0·······|\n······|·······|" +
+                   "\n··············|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n······0·······|\n·····/|········|" +
+                   "\n···············|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n······0·······|\n·····/|\······|" +
+                   "\n···············|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n······0·······|\n·····/|\······|" +
+                   "\n·····/·········|\n···············|\n========="),
+                  ("······+-----+\n······|········|\n······0·······|\n·····/|\······|" +
+                   "\n·····/ \······|\n···············|\n=========")]
+        outcur = []
+        wrongguess = []
+        for i in range(0, len(self.currentword)):
+            if self.currentword[i] in self.guessed:
+                outcur.append(str(self.currentword[i]))
+            else:
+                outcur.append("_")
+        if "_" not in outcur:
+            end = "\n·\n:party_popper: Congratulations! You won! To play again, type @hangman new. :party_popper:"
+            self.ended = True
+        for i in range(0, len(self.guessed)):
+            if self.guessed[i] not in self.currentword:
+                wrongguess.append(str(self.guessed[i]))
+        self.output = stages[self.stage] + "\n·\n```" + "  ".join(outcur) + "" \
+            "```\n·\n```Wrong guesses: " + ", ".join(wrongguess) + "```" + end
+    return self
+
+
+class hangmanHandler(object):
```

Comment:
Conventionally class names should be in camel case:

https://www.python.org/dev/peps/pep-0008/#class-names